### ISSUE TITLE
ElixirLS crashes on start

### ISSUE THREAD

**Issue by gjaldon** [JakeBecker/elixir-ls#42]
I'm using MacOS High Sierra and get the ff ouptut:

![Can't start elixirls](https://www.evernote.com/l/ARjYu7-NEBxDeb5o6vRyHqjjvtYY-x71GIYB/image.png)

**Comment by JakeBecker** [comment]
My guess is that it's failing to launch, and the error message it prints doesn't match the expected wire protocol. Try running this command from a terminal: `ERL_LIBS=~/.vscode/extensions/JakeBecker.elixir-ls-0.2.9/elixir-ls-release mix elixir_ls.language_server`

If it's working, it should just wait without printing anything, but I'm guessing it'll print an error message instead.

**Comment by gjaldon** [comment]
That command ran without error so the server is working fine. Any other ideas what's causing the error? Was hoping to use ElixirLS rather than VSCode. Elixir-VSCode works for me btw.

**Comment by liangjacky** [comment]
I'm getting the same problem. 

```
  ERR Header must provide a Content-Length property.: Error: Header must provide a Content-Length property.
    at StreamMessageReader.onData (/Users/ca8h/.vscode/extensions/JakeBecker.elixir-ls-0.2.12/node_modules/vscode-jsonrpc/lib/messageReader.js:200:27)
    at Socket.<anonymous> (/Users/ca8h/.vscode/extensions/JakeBecker.elixir-ls-0.2.12/node_modules/vscode-jsonrpc/lib/messageReader.js:185:19)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at Pipe.onread (net.js:560:20)
```

**Comment by JakeBecker** [comment]
Fixed in #51
